src/main/java/us/kbase/auth2/lib/identity/MfaStatus.java

```diff
@@ -0,0 +1,65 @@
+package us.kbase.auth2.lib.identity;
+
```

Comment:
```suggestion
package us.kbase.auth2.lib.identity;
// TODO CODE move to lib/token

```

Looking at this I don't think the mfa stuff should be tangled up with the identity code, it's really relevant only to tokens. That can be done later though, we'll just add a TODO